photon-client/src/integTest/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverIntegTest.scala

```diff
@@ -363,7 +363,8 @@ class GameTrainingDriverIntegTest extends SparkTestUtils with GameTestUtils with
     val newArgs = mixedEffectSeriousRunArgs
       .copy
       .put(GameTrainingDriver.rootOutputDirectory, outputDir)
-      .put(GameTrainingDriver.outputMode, ModelOutputMode.TUNED)
+      .put(GameTrainingDriver.outputMode, ModelOutputMode.ALL)
```

Comment:
Here we must change the `outputMode` to `ModelOutputMode.ALL`. If we use `ModelOutputMode.TUNED`, we don't have any tuned model with a dummy tuner. Then it returns an empty output model which causes a failure at line 377 below.

Reply:
Actually I am wondering if we can remove this `testHyperParameterTuning` in `GameTrainingDriverIntegTest` since we won't have atlas-ml in photon-ml, all the following lines in this test become non-sense for testing.

Reply:
Probably best to comment it out, since once Atlas is open-sourced the test will be relevant again.

Reply:
@cmjiang Great find! Let's get the ball rolling on atlas now. This LGTM. @joshvfleming Let's merge!

Reply:
Commented the test out and put a comment on this change.